src/Dapr.AzureFunctions.Extension/Triggers/DaprTopicTriggerAttribute.cs

```diff
@@ -16,8 +16,32 @@ namespace Dapr.AzureFunctions.Extension
     public class DaprTopicTriggerAttribute : Attribute
     {
         /// <summary>
-        /// Gets or sets the name of the topic.
+        /// Initializes a new instance of the <see cref="DaprTopicTriggerAttribute"/> class.
+        /// </summary>
+        /// <param name="pubSubName">The pub/sub name.</param>
+        public DaprTopicTriggerAttribute(string pubSubName)
```

Comment:
Same question about whether the `pubSubName` value can be defaulted to "pubsub". If so, it seems we could make it only optional when initializing this attribute.

Reply:
As Yaron mentioned in the other comment thread, the defaulting behavior only applies to the Dapr CLI, so we should not have any defaults here.

Reply:
See answer above, everywhere else we have made this explicit.

Reply:
The reason I was asking was for the examples. I wanted to make sure the sample code that doesn't define any components matched up with the defaults the CLI creates for ease of use. :+1: I think we're good here.